src/allencell_ml_segmenter/prediction/model_input_widget.py

```diff
@@ -309,13 +296,3 @@ def _configure_slots(self) -> None:
         self._bottom_postproc_button.toggled.connect(
             self._bottom_postproc_button_slot
         )
-
-        # connect postprocessing simple threshold slider to slot
-        self._simple_thresh_slider.changed.connect(
```

Comment:
was this functional before?  I thought we had removed or disabled the post processing UI?

Reply:
IIRC we removed it from the prediction view but kept the code around so that we could have a starting point for thresholding in the future

Reply:
The thresholding was never functional, but we had a slider in the UI at one point which was removed